stratify/_vinterp.pyx

```diff
@@ -544,10 +543,6 @@ cdef class _Interpolation(object):
         fz_src = fz_src.astype(np.float64)
 
         # Broadcast the z_target shape if it is 1d (which it is in most cases)
```

Comment:
It's a legacy comment, happy to remove it.

The thing is, that for the use cases of `z_target` being a `1d` array or non-`1d` array, both use the public `interpolate` function, and based on the dimensionality of `z_target`, it calls the appropriate `_Interpolation` method i.e. `_Interpolation.interpolation` or `_Interpolation._interpolate_z_target_nd`

So, I guess I can nuke the comment, and update the doc-string entry to help clarigy. Ideally, we need some proper documentation to go with `stratify` to make what you can and can't do crystal clear, and how you use the API to get `stratfiy` to do what you want.